### PR TITLE
 Updated for F18 Team, Updated Homepage logo, Added Events | Merge to Master

### PR DESCRIPTION
**Pull request by aseel-mhd**
Changes & Additions
===
- Updated team titles, links, and pictures
- Updated homepage logo to the new GWiCS logo
- Added upcoming events with some dates & times TBD, and updated the calendar

### CODE REVIEW

**Comment by De-ma** [comment]
@aseel-mhd Next time just create a branch since it's easier to keep track of everything under one repo

File: about.html [hunk 1]
```diff
@@ -224,52 +189,52 @@ <h4>Natalie Pirch</h4>
...
             </div>
           </div>
-          <div class="col-sm-4">
+            <div class="col-sm-4">
             <div class="team-member">
-              <img class="mx-auto rounded-circle" src="img/team/jovana.jpg" alt="">
-              <h4>Jovana Kusic</h4>
-              <p class="text-muted">2nd Year Rep!</p>
+              <img class="mx-auto rounded-circle" src="img/team/mary.png" alt="">
+              <h4>Mary Bergin</h4>
```
Comment (De-ma) [inline about.html]: Looks weird if Mary doesn't have a photo

File: about.html [hunk 2]
```diff
@@ -224,52 +189,52 @@ <h4>Natalie Pirch</h4>
...
           </div>
-                      <div class="col-sm-4">
+            <div class="col-sm-4 mx-auto text-center">
             <div class="team-member">
-              <img class="mx-auto rounded-circle" src="img/team/kass.jpg" alt="">
-              <h4>Kassidy Marsh</h4>
-              <p class="text-muted">3rd Year Rep!</p>
+              <img class="mx-auto rounded-circle" src="img/team/jade.jpg" alt="">
+              <h4>Jade Legare!</h4>
+              <p class="text-muted">Event Planner!</p>
```
Comment (De-ma) [inline about.html]: Let's just call Jade 'Event Manager' to be consistent with all the other titles

File: events.html [hunk 3]
```diff
@@ -38,10 +39,10 @@
               <a class="nav-link js-scroll-trigger" href="about.html">About</a>
             </li>
             <li class="nav-item">
-              <a class="nav-link js-scroll-trigger" href="events.html" style="color:black;">Events</a>
+              <a class="nav-link js-scroll-trigger" href="events.html">Events</a>
             </li>
             <li class="nav-item">
-              <a class="nav-link js-scroll-trigger" href="resources.html" >Resources</a>
+              <a class="nav-link js-scroll-trigger" href="resources.html" style="color:black;">Resources</a>
```
Comment (De-ma) [inline events.html]: Why'd you change this? This basically showed which page was open and now its misleading and changes resources to black, when you are on events
screen shot 2018-08-12 at 11 42 52 am

File: events.html [hunk 4]
```diff
@@ -51,17 +52,36 @@
...
+              <li><a href="https://ghc.anitab.org" target="_blank">Grace Hopper Celebration of Women in Computing! Sept 26-28, 2018</a> </li>
+              <li><a href="https://www.can-cwic.ca" target="_blank">The ACM Canadian Celebration of Women in Computing! Nov 2-3, 2018</a> </li>
+              <li><a href="http://cusec.net" target="_blank">Canadian University Software Engineering Conference! Date TBD, 2019</a> </li>
+              <li><a href="https://starcon.io" target="_blank">StarCon! Jan 19-20, 2019</a> </li>
+            </ul>
+            
+            <h2 class="section-subheading text-uppercase">Orientation Week!</h2>
+            <ul>
+              <li>Nerdy Trivia! Date TBD</li>
+              <li>Meet and Greet/Computing Fair, Date TBD</li>
```
Comment (De-ma) [inline events.html]: I believe the meet and greet/showcase is happening Tuesday September 4th

Comment (De-ma) [inline events.html]: I believe the meet and greet/showcase is happening Tuesday September 4th

File: resources.html [hunk 5]
```diff
@@ -56,7 +56,7 @@
       <div class="container">
         <div class="row">
           <div class="col-lg-12">
-            <h1 class="section-heading text-uppercase text-center">Resources for GWiCS</h2>
+            <h1 class="section-heading text-uppercase text-center">Resources for GWiCS</h1>
```
Comment (De-ma) [inline resources.html]: good catch 👀

File: events.html (hunk 4 above)
Comment (De-ma) [reply events.html]: and nerdy trivia is sunday September 2nd at 3pm

Comment (aseel-mhd) [reply events.html]: Thanks!

**Pull request merged**